code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/carbon/human/proc/reconstitute_form() //Scree's race ability.in exchange for: No cloning.
+	set name = "Reconstitute Form"
+	set category = "Abilities"
+
+	if(world.time < last_special)
+		return
+
+	last_special = world.time + 50 //To prevent button spam.
+
+	var/confirm = alert(usr, "Are you sure you want to completely reconstruct your form?? This process can take up to thirty minutes, depending on how hungry you are, and you will be unable to move.", "Confirm Regeneration", "Yes", "No")
+	if(confirm == "Yes")
+		var/mob/living/carbon/human/C = src
+		var/nutrition_used = C.nutrition/2
+
+		if(C.species.reviving == 1) //If they're already unable to
+			C << "You are already reconstructing!"
+			return
+
+		if(C.stat == DEAD) //Uh oh, you died!
+			if(C.reagents.has_reagent("nutriment")) //Let's hope you have nutriment in you.... If not
+				C.nutrition -= C.nutrition/2 //Cut their nutrition in half.
+				var/old_nutrition = C.nutrition //Since the game is being annoying.
+				var/time = (500+1/((nutrition_used/100+1)/1300))
+				C.weakened = 10000 //Since it takes 1 tick to lose one weaken. Due to prior rounding errors, you'd sometimes unweaken before regenning. This fixes that.
+				C.species.reviving = 1
+				C.canmove = 0 //Make them unable to move. In case they somehow get up before the delay.
+				C << "You begin to reconstruct your form. You will not be able to move during this time."
+
+				spawn(time SECONDS)
+					C << "<span class='notice'>We have reconstituted our form.</span>"
+					viewers(C) << "<span class='danger'><p><font size=4> [C] suddenly bursts into a gorey mess, a copy of theirself laying in the viscera and blood. What the fuck?!</font> </span>" //Bloody hell...
+					var/T = get_turf(src)
+					new /obj/effect/gibspawner/human/scree(T)
+					var/braindamage = C.brainloss/2 //If you have 100 brainloss, it gives you 50.
+					C.species.reviving = 0
+					C.revive() // I did have special snowflake code, but this is easier.
```

Comment:
This seems to do 'some stuff' that is unexpected. Like if it replaces limbs that were lost that had items on them or something then the items are attached to you forever. Like I lost a leg, and I could then not remove my shoes after it finished. I was also walking slow even though there was no reason to be.